tests/segmentation_data_prep_test.py

```diff
@@ -64,6 +64,7 @@ def prep_object_and_inputs(
 
 
 def prepare_conversion_matrix():
```

Comment:
A bit nitpicky, but I'd suggest passing in the `rng` fixture which lives in `conftest.py` and using the [Generator API](https://numpy.org/doc/stable/reference/random/generator.html) to these tests / test utility functions.

For example,

```python
def prepare_conversion_matrix(rng: np.random.Generator):
	rng.integers(low=0, high = 3, size=(6,4)).clip(0,1)
	...
```

This generates integers from the same seed, so it'll be reproducible on different systems (even though it only runs on linux for now).

Reply:
That's a cool idea, but I think it doesn't work for this particular function since it does not have `test` prepended in the name, which is required for pytest to supply rng as an argument. I tried decorating the function with `@pytest.mark.usefixtures("rng")`, but that also didn't work. So I'd need to change the name of the function to `test_prepare_conversion_matrix` which also isn't a good solution, since the function is a helper function and does not contain tests. Also I reference the function ~10 times in the codebase and would need to change those references too.
Do you have an idea on how to make use of the pytest fixtures in functions that do not contain "test" in its name?